contracts/Acl.sol

```diff
@@ -102,11 +104,34 @@ contract Permissioned is Ownable {
  * @title ACL enables access control and management for every Smart Vault file
  * @author Lukas Lukac, Lightstreams, 11.7.2018
  */
-contract ACL is Permissioned {
+contract ACL is Initializable, Permissioned {
     constructor(address _owner, bool _isPublic) public Permissioned(_owner, _isPublic) {
     }
 
+    function initialize(address relayHub) public initializer {
```

Comment:
[Medium] As far as I can see the method impls of `initialize()`, `acceptRelayedCall()`, `getRecipientBalance()` and `version()` are the same for every pre-funded contract we want to append to the GSN. So, what about having a contract to encapsulate this a prevent the duplicity iver every contract of the implementation?

Reply:
I tried it but it's not worth it bc the inheritance graph is not going to match many use cases. Sometimes you need implement Initialize SC other times only GSNRecipient for the msg.sender() method etc.

Reply:
But as far as I see that is considered a good base line for that default implementation, just in this PR you already have this code duplicated two times